### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by josephworks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
 Magma will be swaping from craftbukkit to a custom made bukkit/spigot/paper implementation to make it easier to update and make compatibility with mods and plugins better overall.
 
-This update will take some time as we are rewriting a lot of the underlying code 
+This update will take some time as we are rewriting a lot of the underlying code.
 
-### Building the sources
+## Deployment
 
-ATM Building is not ready.
+Currently, Magma 1.14.4 **is NOT** buildable. Therefore, there is currently no Magma 1.14.4 installation process.
```
Comment (RednedEpic) [inline README.md]: This is no longer true. You can build Magma by running `/gradlew reobfShadowJar`.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
-Magma is based on Forge and Paper builds, meaning it can run both Craftbukkit/Spigot/Paper-based plugins and forge-based mods.
+Magma is based on Forge and Paper, meaning it can run both Craftbukkit/Spigot/Paper plugins and Forge mods.
 
-We hope to eliminate all issues with craftbukkit forge servers. In the end, we envision a seamless, low lag Magma experience with support for new 1.12+ versions of Minecraft.
+We hope to eliminate all issues with craftbukkit forge servers. In the end, we envision a seamless, low lag Magma experience with support for newer 1.12+ versions of Minecraft.
 
-### 1.14+ Info
+### 1.14.4 Changes
 
 Magma will be swaping from craftbukkit to a custom made bukkit/spigot/paper implementation to make it easier to update and make compatibility with mods and plugins better overall.
```
Comment (RednedEpic) [inline README.md]: swapping

File: README.md [hunk 3]
```diff
@@ -1,34 +1,38 @@
...
+## About
 
-Magma next  generation in hybrid servers.
+Magma is the next generation of hybrid minecraft server softwares.
 
-Magma is based on Forge and Paper builds, meaning it can run both Craftbukkit/Spigot/Paper-based plugins and forge-based mods.
+Magma is based on Forge and Paper, meaning it can run both Craftbukkit/Spigot/Paper plugins and Forge mods.
 
-We hope to eliminate all issues with craftbukkit forge servers. In the end, we envision a seamless, low lag Magma experience with support for new 1.12+ versions of Minecraft.
+We hope to eliminate all issues with craftbukkit forge servers. In the end, we envision a seamless, low lag Magma experience with support for newer 1.12+ versions of Minecraft.
```
Comment (RednedEpic) [inline README.md]: 1.14+?

File: README.md (hunk 1 above)
Comment (josephworks) [reply README.md]: ok, ill fix that

File: README.md (hunk 2 above)
Comment (josephworks) [reply README.md]: wdym?

Comment (RednedEpic) [reply README.md]: Swaping is incorrect spelling, should be swapping.

File: README.md (hunk 3 above)
Comment (josephworks) [reply README.md]: Not for the About-Magma. Magma, in general, is 1.12+. So that should stay the same (even on 1.14)

File: README.md (hunk 2 above)
Comment (josephworks) [reply README.md]: I didn't spell that, but ill fix that

File: README.md (hunk 3 above)
Comment (josephworks) [reply README.md]: @RednedEpic are you in agreement on this?

Comment (RednedEpic) [reply README.md]: Not my call 😅, just leaving some personal suggestions here.

Comment (josephworks) [reply README.md]: ok then

**Pull request merged**